bundle/manifests/quayregistries.crd.yaml

```diff
@@ -1,8 +1,9 @@
+---
```

Comment:
@flavianmissi isnt' this going to break the whole installation? @jonathankingfc I think we need to remove this line from all files, I remember that OLM can't find the CRD if it contains this thingy here.